### ISSUE TITLE
How can I get the loss for each batch ?

### ISSUE THREAD

**Issue by ssiaw12345** [fchollet/keras#2130]
the History callback gives only loss and acc for each epoch, how can I get the loss for each batch ?

**Comment by meanmee** [comment]
```python 
predict = model.predict(batch)
loss = MSE(batch,predict)
```

**Comment by jrieke** [comment]
The example here does exactly what you want: http://keras.io/callbacks/#create-a-callback
